enigma-cli/src/main/java/cuchaz/enigma/command/Main.java

```diff
@@ -48,13 +48,13 @@ public static void main(String... args) throws Exception {
 				throw new CommandHelpException(cmd, ex);
 			}
 		} catch (CommandHelpException ex) {
-			System.err.println(ex.getMessage());
+			ex.printStackTrace();
```

Comment:
Is the stacktrace useful here? I dont think so.

Reply:
Yeah, would clutter the output more than it'd help, I'm going to revert the change